google/cloud/bigtable/examples/opencensus/WORKSPACE

```diff
@@ -0,0 +1,28 @@
+# Copyright 2018 Google LLC
```

Comment:
I think this file is ignored, does the example work without it?  If it does, please remove it.